### PR TITLE
Component template

### PR DESCRIPTION
**Pull request by kdzwinel**
Template for new components + missing dist/js file for 🍩

### CODE REVIEW

**Comment by matzimowski** [comment]
👍

**Comment by vergilius** [comment]
👍

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contribute
 
 - ES6 + SASS
+- use the template: `/components/template`
 - testing: `npm test`
 - developing: `gulp watch`
-- building: `gulp build --prodcution-build`
-
+- building: `gulp build --production-build`
```
Comment (aju) [inline CONTRIBUTE.md]: empty line?

**Comment by aju** [comment]
OK

File: CONTRIBUTE.md (hunk 1 above)
Comment (kdzwinel) [inline CONTRIBUTE.md]: there were two empty lines, now there is one

Comment (aju) [inline CONTRIBUTE.md]: merge it then!

**Pull request merged**